app/web/features/profile/view/Home.tsx

```diff
@@ -59,121 +36,122 @@ interface HomeProps {
 
 export default function Home({ user }: HomeProps) {
   const classes = useStyles();
+  const { t } = useTranslation([GLOBAL]);
 
```

Comment:
Move all these strings/constants to profile namespace - same reason as the other file.